parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -341,16 +440,16 @@ private void readObjectNoData() throws ObjectStreamException {
 
   }
 
-  public static Binary fromByteBuffer(final ByteBuffer value) {
```

Comment:
This is a breaking change. I just tracked down why semver didn't catch it, but we need to add these methods back and deprecate them.